release/models/acl/openconfig-packet-match.yang

```diff
@@ -313,6 +313,18 @@ module openconfig-packet-match {
         "Source IPv4 address prefix.";
     }
 
+    leaf masked-source-address {
+      type oc-inet:ipv4-address;
+      description
+        "Source IPv4 address used with a mask.";
```

Comment:
I would recommend adding more detailed descriptions to these leafs, namely:
1) indicate that `masked-source-address` and `source-mask` are always used together, but mutually exclusive with `source-address`
2) indicate that `source-address` is the preferred method of configuring ACLs as suggested by @robshakir

Reply:
Done in the last commit.

Reply:
Thanks for the update.  I suggested more explicit text